auth/src/android/auth_android.cc

```diff
@@ -309,6 +309,17 @@ void Auth::DestroyPlatformAuth(AuthData* auth_data) {
   }
 }
 
+void LogHeartbeat(Auth* auth) {
+  // Calling the native getter is sufficient to cause a Heartbeat to be logged.
+  JNIEnv* env = Env(auth->auth_data);
+  jobject platform_app = auth->app()->GetPlatformApp();
+  jobject j_auth_impl = env->CallStaticObjectMethod(
+      auth::GetClass(), auth::GetMethodId(auth::kGetInstance), platform_app);
+  util::CheckAndClearJniExceptions(env);
+  env->DeleteLocalRef(platform_app);
```

Comment:
Though it might not be an issue, I would recommend to delete `j_auth_impl` before `platform_app` since auth usually depends on app.  Just to avoid any possible weird GC racing issue.

Reply:
Done.